### DISCUSSION TITLE
AI branch

### DISCUSSION THREAD

**Discussion by OOHehir** [LIFsCode/ELOC-3.0#25] [General]
Hello ED & Fabian,

I've just created a new branch (ai) & will start pushing code to it. 

I'm sure you're aware but just in case...

I've setup a new project '[task](https://github.com/LIFsCode/ELOC-3.0/blob/fbd1ac47d14a668686baf26db00bb704a9dea515/eloc610LowPowerPartition/platformio.ini#L42)' in the platformio.ini file. I'm using visual studio code & its possible with a single click to build different versions of the project. I think it should be straightforward to use ifdef guards to separate the ai code. It means if necessary we can build versions with & without the ai stuff. I'll continue pushing to the ai branch for the moment until its in a reasonable state.

**Comment by EDsteve** [comment]
@OOHehir 
Thanks. That sounds good. 
I just read this and just in case @LIFsCode did not read it yet. He will get notified now :)
Better to use the @ so we get notified.

**Comment by OOHehir** [comment]
@LIFsCode <https://github.com/LIFsCode> OK, will do

**Comment by LIFsCode** [comment]
Sorry I read it, just forgot to answer :(

@OOHehir that's perfect. I would keep all features in separate branches and merge them into master once they are stable
